docs/README.md

```diff
@@ -103,12 +89,12 @@ Adapt the following commands accordingly.
 
 #### Setup AGiXT
 ```
-git clone https://github.com/Josh-XT/AGiXT && cd AGiXT
+git clone https://github.com/Josh-XT/AGiXT
 pip install poetry==1.5.0
 export PYTHON_KEYRING_BACKEND=keyring.backends.null.Keyring
+cd AGiXT
 poetry install --with gpt4free
-playwright install
-cd agixt
+playwright install # TODO needs clearification if correct
```

Comment:
It is correct as far as I know. The playwright install happens separate from the pip install.

Reply:
> It is correct as far as I know. The playwright install happens separate from the pip install.

Please clarify here: https://github.com/Josh-XT/AGiXT/issues/531